src/megaapi_impl.cpp

```diff
@@ -17372,6 +17397,7 @@ MegaTransferDataPrivate::MegaTransferDataPrivate(const MegaTransferDataPrivate *
     this->uploadTags = transferData->uploadTags;
     this->downloadPriorities = transferData->downloadPriorities;
     this->uploadPriorities = transferData->uploadPriorities;
+    this->notificationNumber = transferData->notificationNumber;
```

Comment:
In these cases, we usually use the setter:
`this->setNotificationNumber(transferData->notificationNumber)`